raiden/tests/unit/test_transport.py

```diff
@@ -11,11 +13,25 @@
 slogging.configure(':DEBUG')
 
 
+class IntervalCheck(object):
+    def __init__(self):
+        self.t1 = time.time()
+
+    def inside(self, tolerance):
+        t2 = time.time()
+        assert abs(t2 - self.t1) < tolerance
+        self.t1 = t2
+
+    def stepped(self, span, tolerance):
```

Comment:
I understand what this is doing but I am not sure if `stepped` is the right name for it. Do you think we could put a different one? Or at least add a docstring to the function to easily reflect the function's purpose while skimming the code?